contributing.md

```diff
@@ -69,6 +69,26 @@ guide](https://github.com/agis/git-style-guide).
 
 ## Coding Style
 
+### Documentation
+
+When documenting bugs, deficiencies, future tasks, or noteworthy things in the
+code, we use two keywords that most editors and tools recognize: `FIXME:` and
+`TODO:`. We use `FIXME` for a *known bug* and `TODO` for everything else. The
+subsequent `:` is important for tooling, such as syntax highlighters. Here are
+two examples:
+
+```cpp
+// FIXME: this currently fails on FreeBSD. 
+// FIXME: this algorithms is broken for i < 0.
+```
+
+A typical `TODO` could be:
+
+```python
+# TODO: refactor this code to separate mechanism from policy
+# TODO: add another argument to process user-defined tags
```

Comment:
These should be full sentences: Capitalized at the start and a dot at the end.

Reply:
I agree with your regarding the missing punctuation. However, the capitalization after a colon is less established. I have come across both styles, although predominantly seen lowercase.

Reply:
I can agree to both, caps or not. The dot at the end, however, is more important, as it keeps the sentence short and makes it a statement.

Reply:
Yeah, maybe we can simply leave the capitalization open, since it's gramatically not a clear black-vs-white issue?

Reply:
I agree. Merging this.